### ISSUE TITLE
I use `<script src=". /***.ts" lang="ts"></script>` cannot get the appropriate type hint

### ISSUE THREAD

**Issue by 2234839** [johnsoncodehk/volar#43]
There is no problem if I copy the code from ts to the script tag.
But I would like to put the ts into a separate file is there a solution?

**Comment by 2234839** [comment]
I could use a little trick like this to get the correct type hint, but I'd still really like to be able to just use src to introduce
```
<script lang="ts">
  import Component from ". /_event-search";
  export default Component;
</script>
```

**Comment by johnsoncodehk** [comment]
Thanks for feedback! I can implement this feature, but need to wait, I have some accumulation works.

**Comment by johnsoncodehk** [comment]
I has make a quick fix for that, please try 0.19.7.
